topo_processor/metadata/metadata_validators/metadata_validator_stac.py

```diff
@@ -19,9 +19,14 @@ class MetadataValidatorStac(MetadataValidator):
 
     def get_validator_from_uri(self, schema_uri: str) -> Any:
         if schema_uri not in self.validator_cache:
-            response = urllib.request.urlopen(schema_uri)
-            schema = response.read()
-            self.validator_cache[schema_uri] = jsonschema_rs.JSONSchema.from_str(schema.decode("utf8"))
+            if schema_uri.startswith("http"):
+                response = urllib.request.urlopen(schema_uri)
+                schema = response.read()
+                self.validator_cache[schema_uri] = jsonschema_rs.JSONSchema.from_str(schema.decode("utf8"))
+            else:
+                with open(schema_uri) as file:
+                    schema = json.load(file)
```

Comment:
could load this directly from the file, as you are doing 

Buffer -> json -> rust py to json -> schema

where as you could do

Buffer -> rust buffer to json -> schema